app/controllers/subscriptions_controller.rb

```diff
@@ -17,6 +18,23 @@ def create
     end
   end
 
+  def webhook
+    event_json = JSON.parse(request.body.read)
```

Comment:
Oh we can't use the JSON they send to us, since someone besides Stripe could send this webhook. Instead we should  grab the event id and ask Stripe for the data again via the API.